helm/reana/Chart.yaml

```diff
@@ -28,11 +28,11 @@ keywords:
 type: application
 # Chart version.
 version: 0.8.1
-kubeVersion: ">= 1.13.0-0 < 1.22.0-0"
+kubeVersion: ">= 1.19.0-0 < 1.22.0-0"
```

Comment:
Please update the upper bound to `1.24.0-0`.

Kubernetes 1.23 was already released, and I tested with 1.23.1, and everything is working well :+1: